cypress/e2e/reservations.cy.js

```diff
@@ -0,0 +1,34 @@
+import * as page from '../fixtures/URLs.json';
+import ProfilePage from '../support/page_objects/profilePage';
+
+describe('Reserve and Unreserve item for a user test suite', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('donorLoginEmail'), Cypress.env('loginPassword'));
+  });
+
+  it('user can mark an item as reserved for a user that has requested it', () => {
+    cy.visit(page.profile, { failOnStatusCode: false });
+    ProfilePage.reserveButton('shoes').click();
+    cy.get('button').contains('Confirm').should('be.visible').click();
+    cy.get('.card')
+      .contains('shoes')
+      .parents('li')
+      .find('button')
+      .contains('Unreserve')
+      .should('be.visible');
+    cy.get('p').contains('Reserved').should('be.visible');
+  });
+
+  it('user can remove the reservation status of their item', () => {
+    cy.visit(page.profile, { failOnStatusCode: false });
+    cy.get('p').contains('Reserved').should('be.visible');
```

Comment:
```suggestion
    cy.get('p').contains(/reserved/i).should('be.visible');
```

This needs to be a regex to keep the tests from failing if we alter the copy. That happened in the last merge to dev, which is currently causing your test to fail in the CI, though the functionality is correct.